File: ClasseJogo/classHero.js
class Hero {
    constructor(name, age, heroType) {
        this.name  = name; 
        this.age = age;
        this.heroType = heroType;
    }

    attack(){
        let attackType = "";
        switch (this.heroType) {
            case "mago":
                attackType = "magia";   
                break;
            case "guerreiro":
                attackType = "a espada";   
                break;
            case "monge":
                attackType = "artes marciais";   
                break;
            case "ninja":
                attackType =  "shuriken";   
                break;
        }
        return `O ${this.heroType} atacou usando ${attackType}`
    }
}

let magoMerlin = new Hero("Merlin",130,"mago")
let guerreiroArthur = new Hero("Arthur",15,"guerreiro")
let mongeShaolin = new Hero("Shaolin",30,"monge")
let ninjaDonatello = new Hero("Donatello",20,"ninja")

console.log(magoMerlin.attack())
console.log(guerreiroArthur.attack())
console.log(mongeShaolin.attack())
console.log(ninjaDonatello.attack())

File: ClasseJogo/classHero.py
class Hero():
    def __init__ (self, name, age, heroType): 
        self.name  = name 
        self.age = age
        self.heroType = heroType

    def attack(self):
        attackType = ""
        if self.heroType == "mago":
            attackType = "magia"   
        elif self.heroType == "guerreiro":
            attackType = "a espada"   
        elif self.heroType == "monge":
            attackType = "artes marciais"   
        elif self.heroType =="ninja":
            attackType =  "shuriken"
            
        return f"O {self.heroType} atacou usando {attackType}"
    
magoMerlin = Hero("Merlin",130,"mago")
guerreiroArthur = Hero("Arthur",15,"guerreiro")
mongeShaolin = Hero("Shaolin",30,"monge")
ninjaDonatello = Hero("Donatello",20,"ninja")

print(magoMerlin.attack())
print(guerreiroArthur.attack())
print(mongeShaolin.attack())
print(ninjaDonatello.attack())
